src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -172,22 +170,30 @@ public static synchronized void addPodTemplate(PodTemplate podTemplate) {
   }
 
   protected static PodTemplate setProxyEnvVar(PodTemplate podTemplate) {
-    LOGGER.info("Adding Proxy EnvVar for Template "+podTemplate.getName());
-    List<TemplateEnvVar> envVars = new ArrayList<TemplateEnvVar>();
+    String podTemplateName = podTemplate.getName();
+    LOGGER.info("Adding Proxy EnvVar for Template " + podTemplateName);
+    List envVars = podTemplate.getEnvVars();
+    List ptEnvVars = new ArrayList();
+
     String httpProxy = System.getenv(HTTP_PROXY_ENV_VAR_NAME);
     String httpsProxy = System.getenv(HTTPS_PROXY_ENV_VAR_NAME);
     String noProxy = System.getenv(NO_PROXY_ENV_VAR_NAME);
 
-    if (HTTP_PROXY_ENV_VAR_NAME != null && !HTTP_PROXY_ENV_VAR_NAME.isEmpty()) {
-      envVars.add(new KeyValueEnvVar(HTTP_PROXY_ENV_VAR_NAME, httpProxy));
+    if (httpProxy != null && !httpProxy.isEmpty()) {
+      ptEnvVars.add(new KeyValueEnvVar(HTTP_PROXY_ENV_VAR_NAME, httpProxy));
+      LOGGER.info("Added EnvVar with key "+HTTP_PROXY_ENV_VAR_NAME+" for PodTemplate "+ podTemplateName);
     }
-    if (HTTPS_PROXY_ENV_VAR_NAME != null && !HTTPS_PROXY_ENV_VAR_NAME.isEmpty()) {
-      envVars.add(new KeyValueEnvVar(HTTPS_PROXY_ENV_VAR_NAME, httpsProxy));
+    if (httpsProxy != null && !httpsProxy.isEmpty()) {
+      ptEnvVars.add(new KeyValueEnvVar(HTTPS_PROXY_ENV_VAR_NAME, httpsProxy));
+      LOGGER.info("Added EnvVar with key "+HTTPS_PROXY_ENV_VAR_NAME+" for PodTemplate "+ podTemplateName);
```

Comment:
Add spaces around the  "+"  sign => 
LOGGER.info("Added EnvVar with key " + HTTP_PROXY_ENV_VAR_NAME + " for PodTemplate " +  podTemplateName);